plugins/mumble_plugin_win32_ptr_type.h

```diff
@@ -117,16 +132,25 @@ static bool inline initialize(const std::multimap<std::wstring, unsigned long lo
 	if (!dwPid)
 		return false;
 
-	pModule=getModuleAddr(modname ? modname : procname);
-	if (!pModule) {
+	hProcess=OpenProcess(PROCESS_QUERY_LIMITED_INFORMATION|PROCESS_VM_READ, false, dwPid);
+	if (!hProcess) {
 		dwPid = 0;
+		pModule = 0;
 		return false;
 	}
 
-	hProcess=OpenProcess(PROCESS_VM_READ, false, dwPid);
-	if (!hProcess) {
+	int result = checkProcessIs64Bit(hProcess);
+	if (result == -1) {
+		return false;
```

Comment:
Leaking hProcess?

Reply:
Yes, I think it's better if we create a dedicated handle in `checkProcessIs64Bit()`, so that the global one needs only `PROCESS_VM_READ`.

Reply:
That just gives us another handle that we mustn't leak ;) I would just close the handle instead of leaking it in the error cases.

Reply:
Of course, but in addition to that it's better to use a dedicated handle for `IsWow64Process()`, as it needs a specific permission which we use only one time.